### DISCUSSION TITLE
Does Redash still provide preview Docker image on Docker Hub?

### DISCUSSION THREAD

**Discussion by myonlylonely** [getredash/redash#6377] [General]
In the previous workflow, the CircleCI will automatically build a preview Docker image and push it to Docker Hub.
But this action was deleted during the migration to Github actions. Could we add this back?

**Comment by justinclift** [comment]
We will, but things aren't yet ready for that.  Maybe next week or so? :smile:

**Reply by haydenseitz** [reply]
nice! came to discussions for this answer 🚀

**Reply by myonlylonely** [reply]
Any news about the preview Docker images?

**Reply by justinclift** [reply]
We had it theoretically working a few days ago, but when we tried the upload didn't work.

So, we'll need to dig into that and fix the upload, probably this weekend.

**Reply by blerest** [reply]
Any news about the preview Docker images?

**Reply by justinclift** [reply]
Not yet.

**Reply by guidopetri** [reply]
Looping back on this thread to say that the preview images are now working.

**Reply by myonlylonely** [reply]
Great!
